src/app/proptypes.js

```diff
@@ -0,0 +1,41 @@
+import propTypes from 'prop-types';
+
+export const imagePropTypes = {
```

Comment:
I tried removing the duplication from these proptypes on lines 3-25, but the linting errors since it doesn't understand proptypes out of context.